app/views/users/show.html.erb

```diff
@@ -1,5 +1,37 @@
-<h1>User with bio and three latest posts</h1>
-<h2>Sample Bio</h2>
-<p>Sample post</p>
-<p>Sample post</p>
-<p>Sample post</p>
\ No newline at end of file
+<div class="user no-hover-effect rounded row align-items-center m-5 mb-1 p-2">
+    <%= image_tag(@user.photo, alt: "user image", class: "col-3 user-images") %>
+    <div class="justify-content-start mb-5 col-6">
+        <h1><%= @user.name %></h1>
+    </div>
+    <h6 class="align-items-end col-3">Number of posts: <%= @user.posts_counter %></h6>
+</div>
+
+<div class="p-5">
+    <div class="shadow-white rounded border p-3 mb-5">
+        <h4>Bio</h4>
+        <p><%= @user.bio %></p>
+    </div>
+    <% if @user.posts.length > 0 %>
+        <h2>Recent Posts</h2>
+        <div>
+            <% @user.recent_posts.each do |post| %>
+                <%= link_to user_post_path(@user.id, post.id) do %>
+                    <div class="shadow-white rounded border p-3 my-3 post">
+                        <h4 class="mb-4"><%= post.title %></h4>
+                        <p><%= post.text %></p>
+                        <div class="d-flex w-100 justify-content-end">
+                            <h6>Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></h6>
+                        </div>
+                    </div>
+                <% end %>
+            <% end %>
+        </div>
+    <% else %>
+      <h2 class="text-center text-secondary"><%= @user.name %> has no posts<h2>
+    <% end %>
+    <div class="d-flex justify-content-center">
+      <% if @user.posts.length > 3 %>
+        <%= link_to "See all posts", user_posts_path(@user), class:'see-all-posts-btn p-3' %>
+      <% end %>
+    </div>
```

Comment:
- [ ] As you can see, I have posts but I cannot find the **See All Posts** button on the page. Kindly fix that.

- [ ] **[OPTIONAL]** From the picture below you can see that the posts counter is not being updated. I have 2 posts, but your program does not recognize this and does not update the counter values when creating a post. I strongly advise you fix this issue.

  ![image](https://github.com/matee-safi/rails-blog/assets/36542986/dc3575c4-75d0-42eb-93b7-e44961098a48)